### PR TITLE
Implementing the Script Gemstone Bagger!

### PR DESCRIPTION
**Pull request by JohnnyPlayy**
* **Addressed Issue(s)**:  https://github.com/rathena/rathena/issues/6990

* **Server Mode**:  RE

* **Description of Pull Request**: 

image

Script written at the request of my friend! @Pokye 

Todo:

- [ ] Tested

- [x] Translated

### CODE REVIEW

File: npc/re/merchants/Gemstone_Bagger.txt [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+// The information in this script was taken from kro
+//===== Changelogs: ==========================================
+//= 1.0 First Version. [JohnnyPlayy, Pokye]
+//============================================================
+ 
+//= Gemstone Bagger
+//============================================================
+-	script	#GemstoneBagger	HIDDEN_NPC,{
+	mes "[Gemstone Bagger]";
+	mes "We pack gemstones in bags in the easy to find way.";
```
Comment (aleos89) [inline npc/re/merchants/Gemstone_Bagger.txt:13]:
```suggestion
	mes "We pack gemstones in bags in an easy to find way.";
```

File: npc/re/merchants/Gemstone_Bagger.txt [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+	mes "What would you like to choose?";
```
Comment (aleos89) [inline npc/re/merchants/Gemstone_Bagger.txt:14]:
```suggestion
	mes "What would you like to do?";
```

**Review by aleos89 — COMMENTED** [review]
Just some minor things that sounded strange!

File: npc/re/merchants/Gemstone_Bagger.txt [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+	mes "[Gemstone Bagger]";
+	mes "We pack gemstones in bags in an easy to find way.";
+	mes "What would you like to do?";
+	next;
+	switch(select("Exchange Gemstones:Cancel")) {
+	case 1:
+		mes "[Gemstone Bagger]";
+		mes "I will prepare a bag for gemstones.";
+		close2;
+		callshop "Gemstone_Bagger", 1;
```
Comment (Atemo) [inline npc/re/merchants/Gemstone_Bagger.txt:21]: the flag is ignored for barter shop type
```suggestion
		callshop "Gemstone_Bagger";
```

File: db/re/item_db_usable.yml [hunk 4]
```diff
@@ -49792,6 +49792,30 @@ Body:
...
     Script: |
       getitem "Explosive_Kunai",500;
+  - Id: 100192
+    AegisName: Gemstone_Blue
+    Name: Blue Gemstone Pouch
+    Type: Usable
+    Buy: 20
+    Weight: 250
+    Script: |
+      getitem "Blue_Gemstone",500;
```
Comment (Atemo) [inline db/re/item_db_usable.yml:49802]: item group on Aegis, but I guess it's fine like that too

**Review by Atemo — APPROVED** [review]

**Comment by Atemo** [comment]
Thanks!

**Pull request merged**